src/order/complete_lattice.lean

```diff
@@ -1144,32 +1144,61 @@ variables [complete_lattice α]
 
 /-- An independent set of elements in a complete lattice is one in which every element is disjoint
   from the `Sup` of the rest. -/
-def independent (s : set α) : Prop := ∀ ⦃a⦄, a ∈ s → disjoint a (Sup (s \ {a}))
+def sindependent (s : set α) : Prop := ∀ ⦃a⦄, a ∈ s → disjoint a (Sup (s \ {a}))
```

Comment:
Here's a perhaps controversial alternative to `sindependent`: put this in the `set` namespace just as `independent`, and use dot notation.

Failing that, I'm not a huge fan of `sindependent`, the `sUnion` analogy notwithstanding. Perhaps just `set_independent`? (But this is only an argument for the dot notation, I guess.)

Reply:
I'm not a huge fan of `sindependent` either -- it was kind of a joke. I thought that making the PR would be one way to have the conversation. The argument against dot notation is that I guess other people might want to use `set.independent` to mean something else (e.g linear independence in a vector space).

Reply:
I've gone for set_independent.